src/asm/lexer.c

```diff
@@ -2039,6 +2039,7 @@ static int yylex_RAW(void)
 		 lexer_GetLineNo(), lexer_GetColNo());
 
 	/* This is essentially a modified `appendStringLiteral` */
+	unsigned int parenDepth = 0;
```

Comment:
Why `unsigned int`, and not e.g. `size_t`?

Reply:
No particular reason. Should I change it?

Reply:
Please do.